src/lib/utilities/Popup/popup.ts

```diff
@@ -1,5 +1,6 @@
 import { get, writable, type Writable } from 'svelte/store';
-import type { Middleware, PopupSettings } from './types';
+import type { Placement } from '@floating-ui/dom';
```

Comment:
Real quick though, here's one of the issues that's a no-go for us. We cannot depend on the dependencies being installed by default. We very much want to make all third party dependencies like this (highlight.js, etc) optional and opt in.

By embedding this import inside the `popup.ts` it's no longer optional and will throw errors for users that install Skeleton without this present, even if they're not using popups. We ran into this very same issue with Highlight.js in the past.